lib/app/modules/languages/bindings/language_binding.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:get/get.dart';
+
+import '../controllers/language_controller.dart';
+
+class LanguageBinding extends Bindings{
+  @override
+  void dependencies() {
+    Get.put(
+      LanguageController(),
+      permanent: true,
```

Comment:
Why has this controller been made permanent?

Reply:
to make sure that the LanguageController is available for the entire app duration, thus providing access to translation functionalities across different parts of the app.

Reply:
We can remove the binding and controller since we're moving it to settings. Also, we don't really need this controller to be permanent right? We're only updating the locale when asked by the user.